SPECS/httpd/httpd.spec

```diff
@@ -110,6 +110,16 @@ WantedBy=multi-user.target
 
 EOF
 
+mkdir -p %{buildroot}%{_rpmconfigdir}/macros.d
```

Comment:
Could you move these to a separate file and include it through the `Source` tags? This will follow the way we've been doing that inside `mariner-rpm-macros.spec`. Also, it's a bit easier to read.:)

Reply:
Fixed